internal/system/agent/router.go

```diff
@@ -127,6 +131,22 @@ func metricsHandler(w http.ResponseWriter, r *http.Request) {
 	return
 }
 
+func healthHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	logs.LoggingClient.Debug("health status data requested", "service names", vars)
+
+	list := vars["services"]
+	var services []string
+	services = strings.Split(list, ",")
+
+	ctx := r.Context()
+	send := getHealth(services, ctx)
```

Comment:
- I like how you've followed (for `getHealth()`) the pattern each that was established for `getMetrics()` and `getConfig()` - This makes for code consistency and ease of understanding 👍 
- Even though we _currently_ end up discarding the `error` returned from the respective call each to `getMetrics()` and `getConfig()`, I advise making the flow consistent with the latter two as we may, for example, uncover (or find) a use case where we need to make use of `error`. (See my other note regarding returning a tuple from the func `getHealth()`.